plugins/modules/remote_management/redfish/redfish_config.py

```diff
@@ -375,6 +396,16 @@ def main():
             elif command == "SetHostInterface":
                 result = rf_utils.set_hostinterface_attributes(hostinterface_config, hostinterface_id)
 
+    elif category == "Sessions":
+        # execute only if we find a Sessions resource
+        result = rf_utils._find_sessionservice_resource()
+        if result['ret'] is False:
+            module.fail_json(msg=to_native(result['msg']))
+
+        for command in command_list:
+            if command == "SetSessionService":
+                result = rf_utils.set_session_service(sessions_config)
```

Comment:
From a readability point of view, it is a bit odd that we had `result` before (in line 401), and then we discarded that entirely and create a new result. Better variabe names might help here, but I won't push the point.